build/Targets/GenerateAssemblyInfo.targets

```diff
@@ -19,6 +19,9 @@
     <AssemblyVersionAttribute Include="System.Reflection.AssemblyInformationalVersionAttribute">
       <_Parameter1>$(NuGetPerBuildPreReleaseVersion)</_Parameter1>
     </AssemblyVersionAttribute>
+    <AssemblyVersionAttribute Include="System.Reflection.AssemblyDescriptionAttribute">
```

Comment:
@tmat has discovered that CoreCLR is putting their SHA1 into the AssemblyInformationalVersion.